src/util/records.py

```diff
@@ -546,6 +556,27 @@ def remove_group(cr, xml_id=None, group_id=None):
 
 
 def rename_xmlid(cr, old, new, noupdate=None, on_collision="fail"):
+    """
+    Rename a :term:`external Identifier` of a record.
+
+    A rename cannot happen when the target name already exists on the database. In such
+    cases there are two options to control how this function behaves:
+    - `fail`: raise an exception and prevent renaming
+    - `merge`: rename the external identifier, remove the old one, and *replace*
+    references. See :func:`~odoo.upgrade.util.replace_record_references_batch`
+
+    Note that this function does not remove records, it only updates xml_ids.
```

Comment:
Wraps inside a `note` admonition.
```suggestion
    .. note::
       This function does not remove records, it only updates xml_ids.
```